.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "repo-infra"]
+	path = repo-infra
+	url = git@github.com:ideahitme/repo-infra.git
```

Comment:
@mikkeloscar @ideahitme is going to change it back to `https` and the upstream repo once it's merged.